components/basic-transformers/src/main/java/org/datacleaner/beans/transform/PlainSearchReplaceTransformer.java

```diff
@@ -66,6 +67,10 @@
     @Description("Replace the entire string when the search string is found.")
     boolean replaceEntireString = false;
 
+    @Configured(order = 4)
+    @Description("Case sensitivity will be ignored during searching and replacing. ")
+    boolean ignoreCaseSensitivity = false;
```

Comment:
I'm not really sure if this is consistent with the rest of our settings, but I don't really like negative preferences.

It's not-not case sensitive... So it's case sensitive :)